docs/pr-coding-guidelines.md

```diff
@@ -144,10 +144,18 @@ We do not use frameworks. However we use libraries, such suitcss-components-grid
 
 ### Javascript
 
-The js folder contains two subfolders: `lib` contains all project source files and `vendor` contains all third party libraries. The files out of the two sub folders contain the compiled source code.
+The js folder contains two sub folders: `lib` contains all project source files and `vendor` contains all third party libraries. The files out of the two sub folders contain the compiled source code.
 
 > Note: All code changes should be made to the files in `lib`
 
+#### Linting
+
+For the consistency of our codebase, we use eslint with these rules: https://github.com/webcompat/webcompat.com/blob/master/.eslintrc. We also use Prettier for formatting our code. We have set up two recipes to check and fixe the codebase.
+
+`npm run lint` check all JavaScript files and display the resulting errors. If you get errors, you have two possibilities. You can correct yourself or you can run `npm run fix` which use Prettier to format the code. 
```

Comment:
Suggesting a little bit different phrasing. Feel free to change it as you like or dismiss, if you feel its not fitting. 

```
`npm run lint` checks all relevant JavaScript files and displays, if something needs to be fixed. 

If you get an error displayed, there are two ways to fix it. 
1. You can run `npm run fix` automatically, which is great for small issues like missing spaces or lines in various files. 
2. You can correct it manually as every error message includes the file and line of the error as well as the rule which was violated.
```